retriever/__main__.py

```diff
@@ -132,21 +132,34 @@ def main():
                 check_for_updates(False)
                 print("\n\nScripts downloaded.\n")
 
-            if args.l is None:
-                all_scripts = datasets()
+            if args.l is None and args.k is None:
+                all_scripts = dataset_names()
                 print("Available datasets : {}\n".format(len(all_scripts)))
                 from retriever import lscolumns
-                lscolumns.printls(dataset_names())
+                lscolumns.printls(all_scripts)
             else:
-                all_scripts = datasets(args.l[0])
-                print("Available datasets : {}\n".format(len(all_scripts)))
-                count = 1
-                for script in all_scripts:
-                    print("{}. {}".format(count, script.title))
-                    print(script.name)
-                    print(script.keywords)
-                    print()
-                    count += 1
+                if args.l:
+                    keywords, param_licenses = None, args.l
```

Comment:
This is dependent on what we plan to do with license and keyword search, but I think this would let people narrow down what they're searching for. In addition, I think if they search with license and keyword, say: 
```bash
retriever ls -l CC0-1.0 -k birds
```
we should return the intersection of the two (although that might be a subjective matter). What are your thoughts on that @henrykironde, @adhaamehab ?

Reply:
right I think the intersection of both the results will  be the right action

Reply:
@zhangcandrew  I changed those lines.
I'll wait for @henrykironde opinion about the result of command like
 `retriever ls -l CC0-1.0 -k birds` then update it.

Reply:
Hi @henrykironde 
Have 2 mins to give us your thoughts on that :smile: ?